SQL/0000-00-00-schema.sql

```diff
@@ -72,6 +72,15 @@ CREATE TABLE `language` (
 INSERT INTO language (language_code, language_label) VALUES
     ('en-CA', 'English');
 
+CREATE TABLE `sex` (
+  `SexID` int(10) unsigned NOT NULL AUTO_INCREMENT,
```

Comment:
What purpose does this column have? It is not being used as a foreign key by any other tables and the other column in this table already has a not null/unique constraint and is being used as a foreign key into this table.

Removing it would solve the "what should it be named" debate in the thread below.

Reply:
True, this column is not useful. I don't mind removing it